common/net/default_client.go

```diff
@@ -1,4 +1,4 @@
-package common
+package net
```

Comment:
Do you think it'd be a good idea to move `HttpClientWithEnvironmentProxy` to a `net/http` package and rename it: `http.ClientWithEnvironmentProxy`. I'm thinking we are going to have more and more http code in the future and this might be interesting to split the pure networking things and the http things, just like the go stdlib did.

Reply:
I think it's a premature optimization. We're never going to have as many use cases for our tiny convenience functions as the stdlib does for all of golang.